### DISCUSSION TITLE
Debugging Hermes

### DISCUSSION THREAD

**Discussion by matuskropuch** [expo/router#322] [Q&A]
Hello! I am struggling with setting up a debugger. On SDK 48 Hermes is default so I decided to go with the Flipper as debugger. After a while of experimenting I found out that using expo-router will display "Unmatched route" instead of providing an array to use for debugging from http://127.0.0.1:19000/json/list as is mentioned in Expo Docs https://docs.expo.dev/guides/using-hermes/#troubleshooting. Is there something I can do about it? (e.g. exlude route from router) Thanks for help 🌞

**Comment by tslater** [comment]
I was using the stock chrome debugger. It was working before I installed expo-router. Now none of my files show up. Not sure what the deal it.
